translations/README.spanish.md

```diff
@@ -0,0 +1,217 @@
+* [GraphQL en GitLab integrado con GitLab auth] (https://www.youtube.com/watch?v=m1ChRhRLq7o) (* 4: 05 minutos *)
+* [Panel de control para 10 millones de viajes con ubicación geográfica (PostGIS, Timescale)] (https://www.youtube.com/watch?v=tsY573yyGWA) (* 3: 06 minutos *)
+
+
+## Soporte y solución de problemas
+
+La documentación y la comunidad lo ayudarán a solucionar la mayoría de los problemas. Si ha encontrado un error o necesita ponerse en contacto con nosotros, puede contactarnos a través de uno de los siguientes canales:
+
+* Soporte y comentarios: [Discordia] (https://discord.gg/vBPpJkS)
+* Seguimiento de problemas y errores: [problemas de GitHub] (https://github.com/hasura/graphql-engine/issues)
+* Siga las actualizaciones del producto: [@HasuraHQ] (https://twitter.com/hasurahq)
+* Hable con nosotros en nuestro [chat del sitio web] (https://hasura.io)
+
+Estamos comprometidos a fomentar un ambiente abierto y acogedor en la comunidad. Consulte el [Código de conducta] (code-of-conduct.md).
+
+Si desea informar un problema de seguridad, [lea esto] (SECURITY.md).
+
+## Contribuyendo
+
+Consulte nuestra [guía contribuyente] (CONTRIBUTING.md) para obtener más detalles.
+
+## Activos de marca
+
+Los activos de la marca Hasura (logotipos, la mascota de Hasura, alimentados por insignias, etc.) pueden ser
+encontrado en la carpeta [assets / brand] (assets / brand). Siéntase libre de usarlos en su
+aplicación / sitio web, etc. Estaríamos encantados si agrega el "Desarrollado por Hasura"
+insignia para sus aplicaciones creadas con Hasura. ❤️
+
+<div style = "display: flex;">
+  <img src = "assets / brand / powered_by_hasura_blue.svg" width = "150px" />
+  <img src = "assets / brand / powered_by_hasura_white.svg" width = "150px" />
+</div>
+
+`` `HTML
+<! - Para fondos claros ->
+<a href="https://hasura.io">
+  <img width = "150px" src = "https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_blue.svg" />
+</a>
+
+<! - Para fondos oscuros ->
+<a href="https://hasura.io">
+  <img width = "150px" src = "https://graphql-engine-cdn.hasura.io/img/powered_by_hasura_white.svg" />
+</a>
+`` `
+
+## Licencia
+
+El núcleo GraphQL Engine está disponible bajo la [Licencia Apache 2.0] (https://www.apache.org/licenses/LICENSE-2.0) (Apache-2.0).
+
+Todos los ** otros contenidos ** (excepto aquellos en [`server`] (server), [` cli`] (cli) y
+Los directorios [`consola`] (consola) están disponibles bajo la [Licencia MIT] (LICENCIA-comunidad).
+Esto incluye todo en [`docs`] (docs) y [` community`] (community)
+directorios.
+
+## Traducciones
+
+Este archivo Léame está disponible en las siguientes traducciones:
+
+- [Japonés: jp:] (traducciones / README.japanese.md) (: orar: [@moksahero] (https://github.com/moksahero))
+- [Francés: fr:] (traducciones / README.french.md) (: orar: [@ l0ck3] (https://github.com/l0ck3))
+- [Español: es:] (traducciones / README.spanish.md) (: orar: [@ parammittal16] (https://github.com/parammittal16))
```

Comment:
Estos emojis traducidos no funcionan. Sospecho que esta es una traducción de Google Translate. 

Is this a Google Translate text? Looks like one, you even translated the emojis.